TheFourFifths/consus-client
# [TFF-154] Existing to longterm

### Summary

You can click on a student's cart in the checkout screen and edit the due date of the item. Merging into task branch since people are slow at approving and this task requires longterm checkout

### Checklist


### How to Review

1. Start server on existing-to-longterm-tff154
2. Lookup boaty(111111)
3. Click the due date area, notice hover text?
4. Enter date before today
5. Verify error
6. Click due date on item again
7. Enter date in the future
8. Witness the updated date

### Links

- [TFF-154](https://msoese.atlassian.net/browse/TFF-154)
- TheFourFifths/consus/pull/TBD
